src/main/java/net/obvj/performetrics/util/DurationFormat.java

```diff
@@ -5,7 +5,8 @@
  * <p>
  * Examples:
  * </p>
- * <table summary="Duration format examples">
+ * <table>
+ * <caption>Duration format examples</caption>
```

Comment:
After some brainstorming we decided to move away from HTML table and go with lists, to avoid the need for a caption or a summary.